core/src/main/java/com/microsoft/applicationinsights/internal/jmx/JmxDataFetcher.java

```diff
@@ -72,14 +73,25 @@
                 Collection<Object> resultForAttribute = fetch(server, objects, attribute.attribute);
                 result.put(attribute.metricName, resultForAttribute);
             } catch (Exception e) {
-                logger.warn("Failed to fetch JMX object '{}' with attribute '{}': ", objectName, attribute.attribute);
-                throw e;
+                //Is it ok if we print out the exception text instead of throwing the exception so we can continue the loop?
+                //Is it ok to print out and log the exception once per metric so that we do not flood the logs?
```

Comment:
yes and yes

Reply:
Thanks :) 
Removed the comments as they are not needed